### ISSUE TITLE
Is it possible to provide an AsciiDoc output option?

### ISSUE THREAD

**Issue by clauswilke** [rstudio/bookdown#548]
It would be very helpful if bookdown had an AsciiDoc output option, since several publishers take manuscripts in AsciiDoc format, and because it can easily be converted into DocBook, which is widely used in the publishing industry.

AsciiDoc is a supported pandoc output option, so adding support for it in bookdown should be possible.

**Comment by yihui** [comment]
Yes, in theory, it is possible; knitr has already supported AsciiDoc several years ago, but I feel it is not a trivial project for bookdown. I will be busy till after this summer, so I guess this won't happen anytime soon, but thanks for the suggestion!

**Comment by coatless** [comment]
Any thoughts as to whether `AsciiDocs` for `bookdown` will be supported in the near future? Or, what can be done to move the needle in that direction?
